src/Compiler/Driver/GraphChecking/TypeCheckingGraphProcessing.fs

```diff
@@ -14,14 +14,16 @@ open System.Threading
 /// <param name="deps">Direct dependencies of a node</param>
 /// <param name="transitiveDeps">Transitive dependencies of a node</param>
 /// <param name="folder">A way to fold a single result into existing state</param>
+/// <param name="sortResultsToAdd">...</param>
```

Comment:
Missing description :)

Also, maybe the parameter should rather be `compareResultsToAdd`? 

(And also, should these docs rather be in the signature file?)